pandas/core/frame.py

```diff
@@ -4977,6 +4977,7 @@ def sort_values(
         inplace=False,
         kind="quicksort",
         na_position="last",
+        key=None
```

Comment:
Can you add a type hint for this?

Reply:
Would you just like `key : typing.Callable`?